package com.guminteligencia.ura_chatbot_ia.infrastructure.exceptions;

public class DataProviderException extends RuntimeException {
    public DataProviderException(String message, Throwable cause) {
        super(message, cause);
    }
}
